internal/redispool/ratelimit.go

```diff
@@ -0,0 +1,163 @@
+package redispool
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+var (
+	tokenBucketGlobalPrefix            = "v2:rate_limiters"
+	bucketCapacityConfigKeySuffix      = "config:bucket_capacity"
+	bucketReplenishmentConfigKeySuffix = "config:bucket_replenishment_interval_seconds"
+)
+
+type RateLimiter interface {
+	// GetTokensFromBucket gets tokens from the specified rate limit token bucket.
+	// bucketName: the name of the bucket where the tokens are, i.e. github.com:api_tokens
```

Comment:
`i.e. or e.g.`? I vote for the second 😁 
Same question for `SetTokenBucketReplenishment` and `getRateLimiterKeys` docs.
```suggestion
	// bucketName: the name of the bucket where the tokens are, e.g. github.com:api_tokens
```

Reply:
im not so fluent in latin, I think you're right.